### PR TITLE
Add changelog for lambdas returning ref

### PR DESCRIPTION
**Pull request by BorisCarvajal**
from #9558

### CODE REVIEW

File: changelog/funcliteralsbyref.dd [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Function literals can now return by reference
+
+Prior to this release, there was no way to declare a function that would return a reference by using an anonymous function.
+$(BR)(It was possible to return a pointer but it's not worth mentioning.)
```
Comment (JinShil) [inline changelog/funcliteralsbyref.dd]: it's --> it was

File: changelog/funcliteralsbyref.dd [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Here this function return by value:
```
Comment (JinShil) [inline changelog/funcliteralsbyref.dd]: return --> returns

**Review by JinShil — APPROVED** [review]

File: changelog/funcliteralsbyref.dd (hunk 1 above)
Comment (BorisCarvajal) [reply changelog/funcliteralsbyref.dd]: "that's" should work too but it seems so emphatic

**Review by jacob-carlborg — APPROVED** [review]
LGTM. I let someone else decide on the exact details of the wording.

**Review by wilzbach — APPROVED** [review]

**Comment by wilzbach** [comment]
The exact wording can still be improved, but as the 2.086 is right across the street, I will go ahead and merge this.

**Comment by wilzbach** [comment]
Thanks a lot again @BorisCarvajal!

**Pull request merged**